modules/ROOT/assets/attachments/sg.yaml

```diff
@@ -16,7 +16,7 @@ properties:
 
       You can make the developer preview visible by setting the endpoint address.
```

Comment:
Not sure if this is versioned but this mentions Developer Preview. Will this only show in 2.8?